crates/celestia-grpc-client/src/client.rs

```diff
@@ -191,7 +192,7 @@ impl ProofSubmitter for CelestiaIsmClient {
             return Err(IsmClientError::InvalidProof("ISM ID cannot be empty".to_string()));
         }
 
-        self.send_tx(proof_msg, "MsgUpdateZKExecutionISM").await
+        self.send_tx(proof_msg).await
     }
 
     async fn submit_state_inclusion_proof(&self, proof_msg: StateInclusionProofMsg) -> Result<TxResponse> {
```

Comment:
I don't think we need this `ProofSubmitter` trait, do we? I think we should consider removing it.
We can just horse the messages straight at the rpc with the `send_tx` method.

Reply:
Yes, you're right. Not adding any value